omod/src/main/webapp/resources/htmlforms/eid_test_results.xml

```diff
@@ -44,57 +44,27 @@
                     <th>Reason for Testing</th>
                     <th>Lab Location</th>
                     <th>Test Type</th>
-                    <th>Lab Test Number</th>
+                    <th>Sample ID</th>
                     <th>HIV Result</th>
                     <th>Result Date</th>
                 </tr>
             </thead>
             <tbody class="visit-table-body">
-                <tr>
-                    <obsgroup groupingConceptId="$hivTestConstruct">
-                        <td><obs conceptId="$hivSampleDate"/></td>
-                        <td><obs conceptId="$testingReason" answerConceptIds="$exposed,$presumed,$unknown"/></td>
-                        <td><obs conceptId="$labLocation"/></td>
-                        <td><obs conceptId="$hivTestType" answerConceptIds="$hivPCR,$hivRapid"
-                                 answerLabels="DBS,RT" style="radio"/></td>
-                        <td><obs conceptId="$labTestNumber" /></td>
-                        <td><obs conceptId="$hivResult"
-                                 answerConceptIds="$negative,$positive,$inconclusive"
-                                 answerLabels="Neg,Pos,Inc"
-                                 style="radio" /></td>
-                        <td><obs conceptId="$hivResultDate" /></td>
-                    </obsgroup>
-                </tr>
-                <tr>
+                <tr class="visit-table-row">
                     <obsgroup groupingConceptId="$hivTestConstruct">
-                        <td><obs conceptId="$hivSampleDate"/></td>
-                        <td><obs conceptId="$testingReason" answerConceptIds="$exposed,$presumed,$unknown"/></td>
+                        <td class="nowrap visit-date"><encounterDate/></td>
+                        <td><obs conceptId="$testingReason"/></td>
                         <td><obs conceptId="$labLocation"/></td>
                         <td><obs conceptId="$hivTestType" answerConceptIds="$hivPCR,$hivRapid"
                                  answerLabels="DBS,RT" style="radio"/></td>
                         <td><obs conceptId="$labTestNumber" /></td>
                         <td><obs conceptId="$hivResult"
-                                 answerConceptIds="$negative,$positive,$inconclusive"
+                                 answerConceptIds="$negative,$positive,$inconclusive,$notDone"
                                  answerLabels="Neg,Pos,Inc"
```

Comment:
Presumably "notDone" should have an answerLabel too?

Reply:
Good catch... I'll add one.